Apps/WebClient/src/ClientApp/src/components/modal/verifySMS.vue

```diff
@@ -85,6 +88,28 @@ export default class VerifySMSComponent extends Vue {
         });
     }
 
+    private getTimeout() {
+        if (this.smsVerificationSent) {
+            let timeUntilResend = moment(
+                this.smsResendDateTime!.toJSDate()
+            ).add(5, "minutes");
+            /* Add 1 for readaiblity sake (starts at 5 minutes and ends at 1 minute remaining, rather than 4 to 0) */
+            this.timeoutMinutes =
+                timeUntilResend.diff(moment.utc(), "minutes") + 1;
+            this.countdownRemaining = this.timeoutMinutes;
+            this.runTimer();
+        }
+    }
+
+    private runTimer() {
+        if (this.countdownRemaining > 1) {
+            setTimeout(() => {
+                this.countdownRemaining -= 1;
+                this.runTimer();
```

Comment:
this looks recursive, does it add a lot of timeouts?

Reply:
removed